engine/base/hash-map.lisp

```diff
@@ -3,20 +3,20 @@
 (defclass hash ()
   (internal))
 
-(defun make-hash (&optional pairs)
+(defun hash-make (&optional pairs)
```

Comment:
`Make-hash` is fine, since its a constructor.
All CL functions also follow the convention:
`make-list`, `make-array`, `make-symbol`, etc.